README.md

```diff
@@ -10,7 +10,7 @@ The project's goal is to analyze potentially malicious documents to improve user
 - OS: Linux or Windows
 
 ## GENERAL
-- Some testdata files contain malicious code! Hence, we encoded them in base64 as a safety measure. Nevertheless, be very careful when opening / processing testdata files!
+- Some testdata files contain malicious code! Hence, we use a xor-encoding (key = 0x42) for some testdata files as a safety measure. Hence, be very careful when opening / processing them! For convenience, we provide a python script (["testdata_encode.py"](https://github.com/google/maldoca/testdata_encode.py)) to encode / decode those files. Example usage: python testdata_encode.py maldoca/service/testdata/c98661bcd5bd2e5df06d3432890e7a2e8d6a3edcb5f89f6aaa2e5c79d4619f3d.docx
 
```

Comment:
can you add  little more on the example usage to indicate where does the output go?  And remind the user that if you encode 2x you get back to the original so you use the same program to encode and decode.

Reply:
done :)